demomultistoreform/Resources/install.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS PREFIX_content_block (id_content_block INT AUTO_INCREMENT NOT NULL, title VARCHAR(255) NOT NULL, description LONGTEXT NOT NULL, enable TINYINT(1) NOT NULL, PRIMARY KEY(id_content_block)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB;
```

Comment:
Should we force the COLLATE and ENGINE or let mysql use default settings?

Wdyt @PierreRambaud

Reply:
Better force it as you can have multiple fields with different collation :sweat_smile: